test/unit/testSoftStops.js

```diff
@@ -76,7 +76,7 @@ describe('Soft Stops', function () {
     describe('Link placeholder replacements', function () {
         it('Filters out link URL placeholders from content', function () {
             const stopPages = {
-                noWill: {placeHolders: ['applicationFormPA1A', 'guidance', 'registryInformation']},
```

Comment:
Do we want to remove `guidance` and `registryInformation` as we still have these in the stoppage content for `noWill`

Reply:
No, see above, this PR includes 3 tickets and PRI-59 is about a copy change on the stop page, where those two are no longer there.